charts/opentelemetry-collector/Chart.yaml

```diff
@@ -11,4 +11,4 @@ maintainers:
   - name: dmitryax
   - name: TylerHelmuth
 icon: https://opentelemetry.io/img/logos/opentelemetry-logo-nav.png
-appVersion: 0.80.0
+appVersion: 0.81.0
```

Comment:
Please revert this back to 0.80.0. We like to do the appVersion bumps in a standalone PR.  I also see the Collector chart's Contributing Guide needs some work to be clearer which I'll fix up.

Reply:
I got the appVersion bumped to `0.81.0`, so now we need a rebase.  Will need to change the chart's version to `0.62.1` and rerun example generation.

Reply:
Sure @TylerHelmuth  there you go

Reply:
I will change straight away